specs/electra/beacon-chain.md

```diff
@@ -941,6 +899,21 @@ def process_effective_balance_updates(state: BeaconState) -> None:
             validator.effective_balance = min(balance - balance % EFFECTIVE_BALANCE_INCREMENT, EFFECTIVE_BALANCE_LIMIT)
 ```
 
+### Execution engine
+
+#### Request data
+
+##### Modified `NewPayloadRequest`
+
+```python
+@dataclass
+class NewPayloadRequest(object):
+    execution_payload: ExecutionPayload
+    execution_requests: ExecutionLayerRequests
+    versioned_hashes: Sequence[VersionedHash]
+    parent_beacon_block_root: Root
```

Comment:
Needs a `# [New in Electra]` comment. And though it makes sense in that location, I'm thinking `execution_requests` should be at the bottom. This makes it easier for clients to extend. What do others think?